test/Microsoft.TemplateEngine.Cli.UnitTests/TemplateSearchCacheTests.cs

```diff
@@ -191,7 +191,7 @@ public async Task CacheSearchLanguageMismatchFilterTest()
             Tags = new Dictionary<string, ICacheTag>(StringComparer.Ordinal)
             {
                 {
-                    "Framework", ResolutionTestHelper.CreateTestCacheTag(new List<string>() { "netcoreapp2.0", "netcoreapp2.1" })
+                    "Framework", ResolutionTestHelper.CreateTestCacheTag(new List<string>() { "netcoreapp2.0", "netcoreapp2.1", "netcoreapp3.1" })
```

Comment:
Should we remove "netcoreapp2.0" from here as well?

Reply:
Yes, probably we should since `netcoreapp2.0` is obsolete, and removing should not affect the tests cache.